src/tests/unit/tests/background/telemetry/telemetry-client-provider.test.ts

```diff
@@ -12,8 +12,8 @@ import { StorageAdapter } from '../../../../../common/browser-adapters/storage-a
 import { configMutator } from '../../../../../common/configuration';
 
 describe('TelemetryClientProvider', () => {
-    beforeEach(configMutator.reset);
-    afterAll(configMutator.reset);
+    beforeEach(() => configMutator.reset());
```

Comment:
This started failing because `this` stopped being bound within the reset call. Frankly, I'm surprised this worked in the first place without an arrow function; some sort of quirk of importing the const object maybe.